.npmignore

```diff
@@ -0,0 +1,4 @@
+**/*
+!index.js
+!lib/**/*
+!src/**/*
```

Comment:
Shouldn't yarn.lock & package.json be added to this list as well?

I think this list is a bit extreme, it's probably a better idea to exclude certain folders than excluding everything except a couple files

Reply:
package.json and README.md are included automatically included. I think taking the opt-in approach guarantees we don't include other files.